packages/server/email/createEmailVerficationForExistingUser.ts

```diff
@@ -0,0 +1,49 @@
+import base64url from 'base64url'
+import crypto from 'crypto'
+import getRethink from '../database/rethinkDriver'
+import AuthIdentityLocal from '../database/types/AuthIdentityLocal'
+import EmailVerification from '../database/types/EmailVerification'
+import {DataLoaderWorker} from '../graphql/graphql'
+import emailVerificationEmailCreator from './emailVerificationEmailCreator'
+import getMailManager from './getMailManager'
+
+const createEmailVerficationForExistingUser = async (
```

Comment:
we already had a helper function to do this with new signups.
this does almost the exact same thing, except it pulls the email & password from the User object instead of from args